service/entrypoint.sh

```diff
@@ -17,5 +17,14 @@ fi
 
 # Run as exec so the application can receive any Unix signals sent to the container, e.g.,
 # Ctrl + C.
-# shellcheck disable=SC2086
-exec java -cp . -server $JAVA_OPTS org.springframework.boot.loader.JarLauncher "$@"
+if [ "${DD_PROFILING_ENABLED:-false}" = "true" ]; then
+  export DD_ENV="${DD_ENV:-invalid}"
```

Comment:
Where is `DD_ENV` defined? Should this use `VOLTTI_ENV` or something?

Reply:
DD_ENV comes environment variables. 

https://docs.datadoghq.com/tracing/setup_overview/setup/java/?tab=containers

We could use VOLTTI_ENV in future insted DD_ENV.

This is still draft to get build to test the metric collection from nodes https://docs.datadoghq.com/agent/amazon_ecs/?tab=awscli that is needed for posting data to datadog.

Reply:
Yeah I was just curious and gave some drive-by comments, no worries 🙂 

`DD_ENV` is supposed to be set by the user, right? Doesn't that line try to assign the current `DD_ENV` value to `DD_ENV` itself, thus always ending up as `invalid`..?

Reply:
Thanks for input. Good points. :slightly_smiling_face: 

`export DD_ENV="${DD_ENV:-invalid}"` is bash trick. So it will set `DD_ENV` to `DD_ENV` when set and if not set then to `invalid`. I changed default value to it to `VOLTTI_ENV` environment variable as you suggested as it might work out of the box without need to define new variable in ecs-definition.